cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -551,20 +602,31 @@ def act_sigmoid_scaled(x):
         checkpoint_callback=ckpt_callback,
     )
 
+    # %%
+    # The Estimator hides the following details:
+    # 1. Binding Spark DataFrames to a deep learning training script
+    # 2. Reading data into a format that can be interpreted by the training framework
+    # 3. Distributed training using Horovod
+    # the user would just provide a Keras model, and the `KerasEstimator`` will fit the data and store it in a Spark DataFrame
```

Comment:
why will the user provide a Keras model? We only provide the data, right?

Reply:
https://horovod.readthedocs.io/en/stable/spark_include.html - that line is here..